generators/client/templates/angular/src/main/webapp/app/layouts/navbar/_navbar.scss

```diff
@@ -10,6 +10,8 @@ Navbar
 .jh-navbar {
     background-color: #353d47;
     padding: .2em 1em;
+    border-radius: 0; /* Hide the navbar rounded corners */
```

Comment:
The `$enable-rounded` not only affects the navbar, but few elements of Bootstrap.
[http://v4-alpha.getbootstrap.com/getting-started/options/](url)
> $enable-rounded : Enables predefined border-radius styles on various components.